qa/tasks/cephfs/test_mirroring.py

```diff
@@ -511,7 +513,14 @@ def test_cephfs_mirror_blocklist(self):
 
         # wait enough so that the mirror daemon restarts blocklisted instances
         time.sleep(40)
-        rados_inst_new = self.get_mirror_rados_addr(self.primary_fs_name, self.primary_fs_id)
+
+        # get the new rados_inst
+        rados_inst_new = ""
+        with safe_while(sleep=1, tries=30, action='wait for mirror status rados_inst') as proceed:
```

Comment:
If safe_while is used then is there a need for the sleep above?

Reply:
Will drop it and check the qa run result.

Reply:
Fixed

Reply:
Test passed: https://pulpito.ceph.com/jcollin-2024-03-19_14:58:23-fs:mirror-main-distro-default-smithi/